### ISSUE TITLE
Add new feature to resource alicloud_ess_scaling_configuration.

### ISSUE THREAD

**Issue by Juan-ijuanix** [terraform-providers/terraform-provider-alicloud#1825]
Hi there,
 I was configured an alicloud_ess_scaling_configuration and I realized that two options that are available in the CLI aliyun and that are very necessary when you deploy windows machines in a scaling group are not available in the provider. These options are:

**Password** | String | No | 123-abcABC | The password   of the ECS instance. The password must be 8 to 30 characters in length. It   must contain at least three of the following character types: uppercase   letters, lowercase letters, digits, and special characters. Special   characters include    ( ) ` ~ ! @ # $ % ^ & * - _ +   = \ \| { } [ ] : ; ' < > , . ? /   The   password of Windows-based instances cannot start with a forward slash (/).   Note For security reasons, we   recommend that you use HTTPS to send requests if the Password parameter is   specified.
-- | -- | -- | -- | --
**PasswordInherit** | Boolean | No | false | Specifies   whether to use the preconfigured password of the specified image. To use this   parameter, ensure that a password is configured for the specified image.

I have successfully implemented one of them PasswordInherit, modifying the source and compiling the provider, I attached the modified files, although I do not know if it is the best way but this woks for me.

It would be very interesting to enable these two crateristics in the next release.

Regards, and thank you very much

[resource_alicloud_ess_scalingconfiguration_test.go.txt](https://github.com/terraform-providers/terraform-provider-alicloud/files/3800287/resource_alicloud_ess_scalingconfiguration_test.go.txt)
[resource_alicloud_ess_scalingconfiguration.go.txt](https://github.com/terraform-providers/terraform-provider-alicloud/files/3800288/resource_alicloud_ess_scalingconfiguration.go.txt)

**Comment by Juan-ijuanix** [comment]
excuse me, I have seen that "password" has already been implemented

**Comment by xiaozhu36** [comment]
HI @Juan-ijuanix Yes, the release 1.60.0 has supported it. Please have a try.

**Comment by Juan-ijuanix** [comment]
Hi @xiaozhu36  Is it PasswordInherit implamented too? I couldn´t find it in the code.

**Comment by xiaozhu36** [comment]
HI @Juan-ijuanix No. Do you need it?

**Comment by Juan-ijuanix** [comment]
Yes, it would be very helpful because I generate images of windows with packer and already have a valid password.

**Comment by xiaozhu36** [comment]
HI @Juan-ijuanix The latest release 1.62.0 has supported it. Please check it.

**Comment by Juan-ijuanix** [comment]
Hi, I found a problem with  PasswordInherit. It seems that the configuration was applied but the modification did not apply to the scaling group, it was as pre configured but not applied. It continued to appear in scaling group configuration as an unconfigured logon. I made these changes in the code and now it works ok, I don't know if it's the best way. 
![ChangeCode](https://user-images.githubusercontent.com/57180335/68896350-913e7880-072b-11ea-84ac-77fdea1ab61d.jpg)

**Comment by xiaozhu36** [comment]
HI @Juan-ijuanix Thanks for your feedback. I am sorry that I can not noticed the parameter can be updated. The PR #1877 has supported it.
